test/linter.js

```diff
@@ -11,7 +11,7 @@ describe('AutoRest Linter validation:', function () {
   // Useful when debugging a test for a particular swagger. 
   // Just update the regex. That will return an array of filtered items.
   // swaggersToProcess = swaggersToProcess.filter(function(item) {
-  //   return (item.match(/.arm-containerregistry.*2017-03-01.*/ig) !== null);
+  //   return (item.match(/.Microsoft.ContainerRegistry.*2017-03-01.*/ig) !== null);
   // });
   _(swaggersToProcess).each(function (swagger) {
     it(swagger + ' should honor linter validation rules.', function (done) {
```

Comment:
@olydis  when running the validator against a configuration file, does the "individual" state run against individual json files or against the composed state?